src/main.go

```diff
@@ -47,7 +46,7 @@ func main() {
 	wg.Add(1)
 
 	go func() {
-		parse.ParseStaticFile("bgptest1", msgChannel)
+		//parse.ParseStaticFile("bgptest1", msgChannel)
```

Comment:
with `ParseStaticFile` commented, this goroutine is basically a harmless no-op. But if that's intentional, you could remove this section completely.